nautobot/dcim/forms.py

```diff
@@ -663,19 +664,35 @@ class Meta:
         ]
 
 
+#
+# Hardware Family
+#
+
+
+class HardwareFamilyForm(NautobotModelForm):
```

Comment:
My thought is that since we are eventually going to remove all the `FilterForm` code and stick to advanced filter UI. Is that still the direction we are going?

Reply:
For now we should still add it until we commit to that plan and implement that.